Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Amazon.Lambda.Annotations.SourceGenerator.csproj

```diff
@@ -17,6 +17,7 @@
   <ItemGroup>
     <!-- Generator dependencies, update Libraries/src/Amazon.Lambda.Annotations.nuspec whenever a new generator dependency is added.  -->
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" GeneratePathProperty="true" PrivateAssets="all" />
+    <PackageReference Include="YamlDotNet" Version="12.0.0" GeneratePathProperty="true" PrivateAssets="all" />
```

Comment:
Cracking open the package from https://www.nuget.org/packages/Amazon.Lambda.Annotations/0.6.0-preview, it looks like this is going to ship inside _our_ package instead of being restored via NuGet?

If so, we need to include the appropriate notice file and track this in the internal attribution tooling as well.

(okay if we punt to a separate PR, since it looks like we need to backfill Netwonsoft too)